spyder/widgets/mixins.py

```diff
@@ -356,6 +366,7 @@ def handle_sub(matchobj):
         return '<br><br>'.join(new_signatures)
 
     def _check_signature_and_format(self, signature_or_text, parameter=None,
+                                    width=_DEFAULT_WIDTH,

```

Comment:
Maybe  `max_width` is a better name as it is more clear what it implies (same as `max_lines`), it could be less than that but never larger. What do you think @dalthviz ?

Reply:
Yep :+1: